### ISSUE TITLE
[Feature] Registering the commands and adding support for the backend server

### ISSUE THREAD

**Issue by RareMaker50** [Sander0542/VelocityUtils#5]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Can execute a command on proxy console, and execute the command through chat in the backend server. Doesn't work executing from backend server console or plugins in the backend server the 

### Expected Behavior

Can execute commands on backend servers and proxy servers, through console and plugins

### Steps To Reproduce

Add plugin, Execute command through the proxy console, Execute the command by you on the backend server. 

Then try to execute via citizens of advanced portals or other plugins on the backend server. 
Then try to execute from console on backend server

Both don't work the backend server 

### Plugin Version

1.1.0

### Velocity Version

3.x

### Java Version

Java 17

### Operating System

Linux

**Comment by Sander0542** [comment]
Since this plugin is only written for Velocity it can only handle commands that are sent or passed thru Velocity (such as chat and proxy console). The backend server do not have access to the Velocity console. This is not a bug, but how the command system in Velocity is written.

**Comment by RareMaker50** [comment]
Yes I get that, there was no feature label so I used bug. I was asking for a feature to be added which is registering the commands on the backend server via the messaging channel
